src/reviewing.md

```diff
@@ -0,0 +1,36 @@
+# Reviewing
+
+Every member of the Library Team, Library API Team, and Library Contributors has 'r+ rights'.
+That is, the ability to approve a PR and instruct [`@bors`](https://bors.rust-lang.org/)
+to test and merge it into Rust nightly.
+
+If you decide to review a PR, thank you!
+But please keep in mind:
+
+- You are always welcome to review any PR, regardless of who it is assigned to.
+  However, do not approve PRs unless:
+    - You are confident that nobody else wants to review it first. If you think someone else on the team would be a better person to review it, feel free to reassign it to them.
+    - You are confident in that part of the code.
+    - You are confident it will not cause any breakage or regress performance.
+    - It does not change the public API, including any stable promises we make in documentation, unless there's a finished FCP for the change.
+      - For unstable API changes/additions, it can be acceptable to skip the RFC process if the design is small and the change is uncontroversial.
+        Make sure to involve `@rust-lang/libs-api` on such changes.
+- Always be polite when reviewing: you are a representative of the Rust project, so it is expected that you will go above and beyond when it comes to the Code of Conduct.
+
+## High-five rotation
+
+Some of the members of the team are part of the 'high-five rotation';
+the list from which the high-five bot picks reviewers to assign new PRs to.
+
+Being a member of one of the teams does not come with the expectation to be on this list.
+However, members of this list should be on at least one of the three library teams.
+
+If the bot assigns you a PR for which you do not have the time or expertise to review it,
+feel free to reassign it to someone else.
+To assign it to another random person picked from the high-five rotation,
+use `r? rust-lang/libs`.
+
+If you find yourself unable to keep up with reviews,
+it might be a good idea to (temporarily) remove yourself from the list.
```

Comment:
I think this may be counterproductive guidance. At the moment, among the reviewers in the "libs" highfive group I have 18 outstanding assigned PRs, m-ou-se has 34, yaahc has 36, joshtriplett has 52, kennytm has 26, Mark-Simulacrum has 29 &mdash; so I don't consider it the case that any of us are "keeping up with reviews". The reality is there is enormous elasticity in incoming reviews: if we had 10&times; the review bandwidth, way more people would see standard library as an awesome contribution opportunity and easily send 10&times; as many PRs. I think a backlog is inevitable and healthy.

All of us are actively making progress on reviews though (which is different from keeping up). That's the best thing to do, along with vaguely prioritizing PRs that are the best cost/benefit, i.e. deferring PRs that we know will consume disproportionate bikeshedding bandwidth for relatively low impact.

I think **rebalancing** the reviewers' queues is more effective than pruning the reviewer list. That can be something automatic we work out, or just manually grabbing good cost/benefit reviews from someone else's queue if they have many stacked up and you have only poor cost/benefit ones outstanding.

Of course if someone literally isn't doing reviews anymore then it would make sense to get off the list, and maybe that's what is meant by "keeping up" here but it isn't how I understood this section.

For raising the aggregate bandwidth of the review rotation, that's just gonna be up to volunteers being willing to take on the work and companies finding it a good investment. I don't think keeping up is really an objective in the first place.